worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/StorageManager.scala

```diff
@@ -857,9 +859,27 @@ final private[worker] class StorageManager(conf: CelebornConf, workerSource: Abs
         flushOnMemoryPressure(writers)
       }
     })
-    flushOnMemoryPressure(hdfsWriters)
-    flushOnMemoryPressure(s3Writers)
-    flushOnMemoryPressure(ossWriters)
+    if (hdfsWriters.size() > 0) {
```

Comment:
Could you extract common codes to a method?

Reply:
PTAL.